### ISSUE TITLE
Whatsapp share button does not work in mozilla firefox and opera browser

### ISSUE THREAD

**Issue by gundeepsn84** [MurhafSousli/ngx-sharebuttons#386]
#### I am submitting a

- [x] Bug Report

#### What is the expected behavior?
click on whatsapp share button it should open whatsapp web

#### What is the current behavior?
click on whatsapp share button it should open whatsapp web.
It works fine in Google chrome v73 but it does not opens whatsapp web on Mozilla Firefox 65.0.2 and Opera 58.
It shows <strong><i>Firefox doesn’t know how to open this address, because one of the following protocols (whatsapp) isn’t associated with any program or is not allowed in this context.</i></strong>

#### What are the steps to reproduce?
Refer this StackBlitz link
https://stackblitz.com/edit/whatsapp-ngxsahre-issue

#### Which versions are you using for the following packages?

Angular: 7.x
Angular CDK: 7.x
Angular CLI: 7.x
Angular-fontawesome: 0.3.0
Share buttons: 7.1.2

#### Is there anything else we should know?
This only occurs in Mozilla Firefox and Opera, but works fine in Google chrome

**Comment by omijoshi09** [comment]
Sorry but in my case I facing this issue in google chrome also, Please help me

**Comment by LenkaYova** [comment]
Hi @MurhafSousli ! I noticed that Whatsapp share link doesn't work on iPhones. I tried to follow this issue solving from 1 year ago: https://github.com/MurhafSousli/ngx-sharebuttons/issues/208
however .forRoot triggered error "Property 'forRoot' does not exist on type 'typeof ShareButtonsModule'"
For Desktop my Whatsapp share button works. But I'm adding it here as comment because at the end it might be connected.
